_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,238 @@
+regular sections with summaries of notable improvements to clients and
+services, announcements of releases and release candidates, and changes
+to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **MuSig2 paper published:** Jonas Nick, Tim Ruffing and Yannick Seurin
+  published [the MuSig2 paper][musig 2 paper] describing a new variant of the
+  [MuSig][topic musig] signature scheme with a two round signing protocol.
+
+    MuSig is a signature scheme that allows multiple signers to create an
+    aggregate public key derived from their individual private keys, and then
+    collaborate to create a single valid signature for that public key. The
+    aggregate public key and signature are indistinguishable from any other
+    [schnorr][topic schnorr signatures] public key and signature.
+
+    The original version of MuSig required a three round signing protocol:
+    first, the co-signers exchange commitments to nonce values, then they
+    exchange the nonce values themselves, and finally they exchange the
+    partial signatures. This three round protocol is to avoid nonce reuse and
+    related attacks, which could allow a dishonest signer to discover the private
+    key of one of their co-signers. Using deterministic nonces, which is very
+    common practice in single-signer schemes, is unsafe for multi-signer schemes,
+    as described in the [original MuSig blog post][musig blog post unsafe
+    deterministic nonce].  Precomputing the nonces in advance and exchanging them
+    at key setup time is also unsafe, as described in a [blog post by Jonas
+    Nick][unsafe nonce sharing]. It _is_ however safe to precompute the nonces and
+    exchange the nonce commitments early, moving one of the three rounds to the key
+    setup stage.
+
+    Removing the requirement to exchange nonce commmittments has been an active
+    area of research, and last month the [MuSig-DN paper][musig-dn] was published, demonstrating
+    how nonce commitment exchange could be removed by generating the nonce
+    deterministically from the signers' public keys and the message, and providing a
+    non-interactive zero-knowledge proof that the nonce was generated
+    deterministically along with the nonce. This removed the requirement of
+    exchanging nonce commitments at the cost of a more expensive signing operation.
+
+    The MuSig2 scheme achieves a two round signing protocol without the need for a
+    zero-knowledge proof. What's more, the first round (nonce exchange) can be done
+    at key setup time, allowing two different variants:
+
+    - interactive setup (key setup and nonce exchange), and non-interactive signing
+    - non-interactive setup (address computed from public keys) and interactive
+      signing (nonce exchange followed partial signature exchange)
+
+    The non-interactive signing variant could be particularly useful for cold storage
+    schemes and offline signers.
```

Comment:
Another prominent potential use case is LN, where you could exchange nonces already when you setup a channel.

Reply:
Added. Thanks!